cli.py

```diff
@@ -15,7 +15,7 @@
 @click.option('--resume_verify', is_flag=True, help="Resume verification of the existing OSF static mirror, need to "
                                                     "import task file")
 @click.option('--delete', is_flag=True, help="Delete nodes from the mirror that have been deleted by users. Requires "
-                                            "compile_active-produced active-node taskfile")
+                                             "'compile_active-produced active-node taskfile' ")
```

Comment:
@adlius Not that, the single quotes around the 'compile_active-produced active-node taskfile'. The single quotes will show up in the CLI like this: 
![screen shot 2016-08-10 at 1 28 40 pm](https://cloud.githubusercontent.com/assets/15851093/17563665/6675a44e-5efe-11e6-939f-55c1896acf8e.png)